docs/faq/nuclei.md

```diff
@@ -21,29 +21,29 @@
 
 ??? info "What kind of scans can I perform with nuclei?"
 
-	Nuclei can be used to detect security vulnerabilities in **Web Applications**, **Networks**, **DNS** based misconfiguration, and **Secret scanning** in source code or files on the local file system.
+	Nuclei can be used to detect security vulnerabilities in **Web Applications**, **Networks**, **DNS** based misconfiguration, and **Secrets scanning** in source code or files on the local file system.
 
 ??? info "How well-maintained is this project?"
 
-	Nuclei project is developed and maintained by [ProjectDiscovery](https://projectdiscovery.io/#/) team and is in active development (we tend to push release every other week).
+	The Nuclei project is developed and maintained by [ProjectDiscovery](https://projectdiscovery.io/#/) team and is in active development (we generally release every other week).
 
 ??? tip "How can I support/contribute to this project? 💙"
 
-	To keep us motivated to work on this project, we request you to write/share new nuclei templates with the community at [template project](https://github.com/projectdiscovery/nuclei-templates) and help us to maintain this public and ready to use / up-to-date nuclei templates.
+	To keep us motivated to work on this project, we request you to write/share new nuclei templates with the community at [template project](https://github.com/projectdiscovery/nuclei-templates) and help us to maintain this public and ready to use, up-to-date nuclei templates.
 
-	If you found an interesting/unique security issue using nuclei and wanted to share the process walk-through with everyone in the form of a blog, we are happy to publish your guest blog at https://blog.projectdiscovery.io.
+	If you found an interesting/unique security issue using nuclei and want to share the process walk-through with everyone in the form of a blog, we are happy to publish your guest blog at https://blog.projectdiscovery.io.
 
 ??? warning "I found results with nuclei. When should I report it?"
 
-	**Wait for a moment**, after nuclei detected security issue, it's always advised to have a second look before reporting it. Here is a tip to confirm/validate the found matches.
+	**Wait a minute** -- after nuclei detected a security issue, it's always advised to have a second look before reporting it. Here's a tip to confirm/validate the found matches.
```

Comment:
There is a chance that no everyone is familiar with the "matcher" term, hence we should rather say "**findings**" or "**found issues**" instead of "**found matches**"